httpie/output/streams.py

```diff
@@ -70,23 +71,28 @@ def build_output_stream(args, env, request, response, output_options):
     output = []
     Stream = get_stream_type(env, args)
 
+    write_info('build_output_stream', 'Total: 4')
     if req:
+        has_branched('build_output_stream', 1)
         output.append(Stream(
             msg=HTTPRequest(request),
             with_headers=req_h,
             with_body=req_b))
 
     if req_b and resp:
```

Comment:
Don't ``add`` in if's count as two?

Reply:
Do you mean `and`'s?
I don't know. Depends on how deep you want to go. If we do that we might be able to write more tests since it's more specific than the coverage measured on coveralls

Reply:
Don't know, we did that on the first one (``_guess_method()``) so I thought we would follow that?